frontend/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "dev": "vite",
+    "dev": "vite --host",
```

Comment:
what does this do?

Reply:
The --host option allows the server to be accessed over the network, not just localhost. it's necessary when running via docker.
When running without --host and exposing the port from the container to the local machine we weren't able to see the output. With that argument, the Frontend was exposed.